.ci/tweet-releasenotes.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e
+
+mkdir .ci-temp
+if [ -d .ci-temp/contribution ]; then
+  cd .ci-temp/contribution/
+  git reset --hard origin/master
+  git pull origin master
+  git fetch --tags
+  cd ../../
+else
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/contribution
+  cd ../
+fi
+
+cd .ci-temp/contribution/releasenotes-builder
+mvn -e --no-transfer-progress clean compile package
+cd ../../../
+
+if [ -d .ci-temp/checkstyle ]; then
+  cd .ci-temp/checkstyle/
+  git reset --hard origin/master
+  git pull origin master
+  git fetch --tags
+  cd ../../
+else
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/checkstyle
+  cd ../
+fi
+
+cd .ci-temp/checkstyle
+LATEST_RELEASE_TAG=$(git describe $(git rev-list --tags --max-count=1))
+cd ../../
+
+CS_RELEASE_VERSION=$(mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
+              -Dexec.args='${project.version}' \
+              --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec | sed 's/-SNAPSHOT//')
```

Comment:
there is a functions in util.sh that extracts checkstyle version and clone from repo. I believe they can be reused here

Reply:
Yes, that's a good idea. Changes Done.

Reply:
what about `checkout_from` function?

Reply:
Right. I am adding.